src/components/ha-selector/ha-selector-media.ts

```diff
@@ -0,0 +1,281 @@
+      ></ha-entity-picker>
+      ${this._manual || !supportsBrowse
+        ? html`${!supportsBrowse
+              ? html`<ha-alert>
+                  ${this.hass.localize(
+                    "ui.components.selectors.media.browse_not_supported"
+                  )}
+                </ha-alert>`
+              : ""}
+            <ha-form
+              .hass=${this.hass}
+              .data=${this.value}
+              .schema=${MANUAL_SCHEMA}
+              .computeLabel=${this._computeLabelCallback}
+            ></ha-form>`
+        : html`<ha-card
+            outlined
+            @click=${this._pickMedia}
+            class=${this.disabled || !this.value?.entity_id ? "disabled" : ""}
+          >
+            <div
+              class="thumbnail ${classMap({
+                portrait:
+                  !!this.value?.metadata?.media_class &&
+                  MediaClassBrowserSettings[
+                    this.value.metadata.children_media_class ||
+                      this.value.metadata.media_class
+                  ].thumbnail_ratio === "portrait",
+              })}"
+            >
+              ${this.value?.metadata?.thumbnail
+                ? html`
+                    <div
+                      class="${classMap({
+                        "centered-image":
+                          !!this.value.metadata.media_class &&
+                          ["app", "directory"].includes(
+                            this.value.metadata.media_class
+                          ),
+                      })}
+                        image"
+                      style=${this._thumbnailUrl
+                        ? `background-image: url(${this._thumbnailUrl});`
+                        : ""}
+                    ></div>
+                  `
+                : html`
+                    <div class="icon-holder image">
+                      <ha-svg-icon
+                        class="folder"
+                        .path=${!this.value?.media_content_id
+                          ? mdiPlus
+                          : this.value?.metadata?.media_class
+                          ? MediaClassBrowserSettings[
+                              this.value.metadata.media_class === "directory"
+                                ? this.value.metadata.children_media_class ||
+                                  this.value.metadata.media_class
+                                : this.value.metadata.media_class
+                            ].icon
+                          : mdiPlayBox}
+                      ></ha-svg-icon>
+                    </div>
+                  `}
+            </div>
+            <div class="title">
+              ${!this.value?.media_content_id
+                ? this.hass.localize("ui.components.selectors.media.pick_media")
+                : this.value.metadata?.title || this.value.media_content_id}
+            </div>
+          </ha-card>`}
+      ${supportsBrowse
+        ? html`<mwc-button @click=${this._switchManual}
```

Comment:
I don't like this button. I think that if we want to offer this, it should be part of the "pick media" dialog instead.

Reply:
We could make a special navigation ID called `manual://` that will show this form

Reply:
You also want to do that for entities not supporting media browser?

Reply:
yeah

Reply:
We can do that in a future PR if you want.